src/components/report-viewer/log-table.tsx

```diff
@@ -120,7 +120,7 @@ const LogTable = ({ selectedReport, reportType, severities, onRowClick }: LogTab
     }, [severityFilter, fetchReportLogs, selectedReport, messageFilter, resetSearch]);
 
     useEffect(() => {
-        if (filters?.length === 0 && severities && severities.length > 0) {
```

Comment:
Removing the condition altogether has some side effects in other use cases. it resets the displayed severities selection when switching selected nodes from the treeview

Reply:
:+1: 
Fixed